README.md

```diff
@@ -87,6 +87,7 @@ The OpenFGA team will do its best to address all production issues with high pri
 
 # Contributing
 
+See [CONTRIBUTING](./CONTRIBUTING.md).
```

Comment:
I think this is a broken link. Should this link to `CONTRIBUTING.md` in our `.github` repo?